src/m365/planner/commands/task/task-add.ts

```diff
@@ -213,6 +213,9 @@ class PlannerTaskAddCommand extends GraphCommand {
 
       const newTask = await request.post<PlannerTask>(requestOptions);
       const result = await this.updateTaskDetails(args.options, newTask);
+      if (result.description) {
+        result.hasDescription = true;
+      }
```

Comment:
This is a quick workaround for the current issue but I'm wondering if it would be interesting to expand this logic by including a final get call to fetch the latest status of our task item. When we get the task item again from Graph, you'll see that the property `hasDescription` will be updated properly. We should keep in mind that this extra call only needs to be done when we are executing the method `updateTaskDetails`, so this would be when the users execute the command with the options `description` or `previewType`.

What are your thoughts @nicodecleyre, @milanholemans, and @pnp/cli-for-microsoft-365-maintainers?

Reply:
This was also my first thought, and the most future proof solution. However it will significantly increase our execution time of course.
I can live with both approaches.

Reply:
I'm with Milan. the extra call is most future proof but it is still an extra call

Reply:
For the time being, let's leave it as is and continue with this approach 😄